ansible/roles/ros_workspace/tasks/main.yml

```diff
@@ -58,6 +58,10 @@
   with_items: repo_host_keys.results
   when: repo_host_keys.stdout is defined repo_host_keys.stdout is not ""
 
+- name: Delete any files in {{ros_workspace}}/src to have a fresh workspace
```

Comment:
I had some issue of wstool up failing - deleting the src is a bit drastic but does work